modules/installation-proxy-configure-object.adoc

```diff
@@ -0,0 +1,121 @@
+
+[id="installation-proxy-configure-object_{context}"]
+= Enabling cluster-wide proxy
+
+The proxy object is used to manage cluster-wide egress proxy. When a cluster is
+installed or upgraded without proxy configured, a proxy object is still
+generated but it will have a nil spec and status. For example:
+
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Proxy
+metadata:
+  creationTimestamp: "2019-08-21T22:36:49Z"
+  generation: 2
+  name: cluster
+  resourceVersion: "24913"
+  selfLink: /apis/config.openshift.io/v1/proxies/cluster
+  uid: 2a344b01-c464-11e9-b3f3-025de4b59c38
+spec:
+  trustedCA:
+    name: ""
+status:
+----
+
+A cluster administrator can configure proxy for {product-title} by modifying
+this `cluster` proxy object.
+
+NOTE: Only the proxy object named `cluster` is supported, and no additional
+proxies can be created.
+
+.Prerequisites
+
+* Cluster administrator permissions
+* {product-title} `oc` CLI tool installed
+
+.Procedure
+
+. Use the `oc edit` command to modify the proxy:
++
+----
+$ oc edit proxy/cluster
+----
+
+. Create a ConfigMap that contains any additional CA certificates required for
+proxying HTTPS connections.
++
+NOTE: You can skip this step if the proxy’s identity certificate is signed by an
+authority from the RHCOS trust bundle.
+
+.. Create a file with the following contents and the values of your PEM-encoded
+certificates:
++
+[source,yaml]
+----
+apiVersion: v1
+data:
+  ca-bundle.crt: |
+    <MY_PEM_ENCODED_CERTS> <1>
+kind: ConfigMap
+metadata:
+  name: user-ca-bundle <2>
+  namespace: openshift-config <3>
+----
+<1> One or more PEM-encoded X.509 certificates used to sign the proxy's
+identity certificate.
+<2> The ConfigMap name that will be referenced from the proxy object.
+<3> The ConfigMap must be in the `openshift-config` namespace.
+
+.. Create the ConfigMap from this file:
++
+----
+$ oc create -f user-ca-bundle.yaml
+----
+
+. Configure the necessary fields for the proxy:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Proxy
+metadata:
+  creationTimestamp: "2019-08-21T22:36:49Z"
+  generation: 2
+  name: cluster
+  resourceVersion: "24913"
+  selfLink: /apis/config.openshift.io/v1/proxies/cluster
+  uid: 2a344b01-c464-11e9-b3f3-025de4b59c38
+spec:
+  httpProxy: http://<username>:<pswd>@<ip>:<port> <1>
+  httpsProxy: https://<username>:<pswd>@<ip>:<port> <2>
+  noProxy: example.com <3>
+  readinessEndpoints:
+  - http://www.google.com <4>
+  - https://www.google.com
+  trustedCA:
+    name: user-ca-bundle <5>
+status:
+  httpProxy: http://<username>:<pswd>@<ip>:<port>
+  httpsProxy: https://<username>:<pswd>@<ip>:<port>
+  noProxy: 10.0.0.0/16,10.128.0.0/14,127.0.0.1,169.254.169.254,172.30.0.0/16,api-int.cluster.example.com,api.cluster.example.com,etcd-0.cluster.example.com,etcd-1.cluster.example.com,etcd-2.cluster.example.com,example.com,localhost
+----
+<1> A proxy URL to use for creating HTTP connections outside the cluster.
```

Comment:
I'd probably

_A proxy URL to use when making HTTP connections with hosts outside the cluster._